config/locales/client.en.yml

```diff
@@ -4825,6 +4825,8 @@ en:
         go_back: "Back to Search"
         recommended: "We recommend customizing the following text to suit your needs:"
         show_overriden: "Only show overridden"
+        locale: "Language:"
+        fallback_locale_warning: "If a translation is missing, the %{fallback} version will be used instead."
```

Comment:
I was thinking of a different warning. Something along the lines of

> You are editing a variety of %{fallback}. Users who choose %{fallback} as their interface language won't see your changes.

Reply:
Thanks for the suggestion, it sounds way better!